WORKSPACE

```diff
@@ -142,19 +142,37 @@ http_file(
 
 http_file(
     name = "fedora_image",
-    sha256 = "423a4ce32fa32c50c11e3d3ff392db97a762533b81bef9d00599de518a7469c8",
+    sha256 = "a30549d620bf6bf41d30a9a58626e59dfa70bb011fd7d50f6c4511ad2e479a39",
     urls = [
-        "https://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2",
-        "https://storage.googleapis.com/builddeps/423a4ce32fa32c50c11e3d3ff392db97a762533b81bef9d00599de518a7469c8",
+        "https://download.fedoraproject.org/pub/fedora/linux/releases/29/Cloud/x86_64/images/Fedora-Cloud-Base-29-1.2.x86_64.qcow2",
```

Comment:
we've switched to fedora 32, You probably shouldn't change this. Or has it been done on purpose?

Reply:
I am only reverting this commit as a whole, in this PR, in order to test migration,
since this PR made guest agent connection problems.
There is a hold on the PR so it wont be merged, once the code is fine, i will revert this commit and remove the lgtm.
There are ongoing PRs by the team to fix the problem fc32 presented meanwhile.

Reply:
reverted